app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    masks_to_buy = 0
+    for visitor in friends:
+        try:
+            try:
+                cafe.visit_cafe(visitor)
+            except NotWearingMaskError:
+                masks_to_buy += 1
```

Comment:
The nested `try` block is unnecessary. You can handle both exceptions in a single `try` block to simplify the code.